File: Web/Model/Task.cs
using System;
using Newtonsoft.Json;

namespace MyIssue.Web.Model
{
    public class Task
    {
        [JsonProperty("TaskId")]
        public decimal TaskId { get; set; }

        [JsonProperty("TaskTitle")]
        public string TaskTitle { get; set; }

        [JsonProperty("TaskDescription")]
        public string TaskDescription { get; set; }

        [JsonProperty("TaskClient")]
        public string TaskClient { get; set; }

        [JsonProperty("TaskAssignment")]
        public string TaskAssignment { get; set; }

        [JsonProperty("TaskOwner")]
        public string TaskOwner { get; set; }

        [JsonProperty("TaskType")]
        public string TaskType { get; set; }

        [JsonProperty("TaskStart")]
        public DateTime? TaskStart { get; set; }

        [JsonProperty("TaskEnd")]
        public DateTime? TaskEnd { get; set; }

        [JsonProperty("TaskCreationDate")]
        public DateTime TaskCreationDate { get; set; }

        [JsonProperty("CreatedByMail")]
        public string CreatedByMail { get; set; }

        [JsonProperty("EmployeeDescription")]
        public string EmployeeDescription { get; set; }
    }
}

File: Web/ClientApp/src/models/task.ts
export interface IPagedTaskRequest {
  link: string,
  page: number,
  size: number,
}
export interface IPagedResponse {
  pageNumber: number,
  pageSize: number,
  firstPage: string,
  lastPage: string,
  totalPages: number,
  totalRecords: number
  nextPage: string,
  previousPage: string,
  data: ITask[]
}
export interface ITaskroot {
  tasks: ITask[];
}
export interface ITask {
  taskId: string,
  taskTitle: string,
  taskDescription: string,
  taskClient: string,
  taskAssignment: string,
  taskOwner: string,
  taskType: string,
  taskStart: string,
  taskEnd: string,
  taskCreationDate: string,
  createdByMail: string,
  employeeDescription: string,
}
